decisions/0009-schema-refactor.md

```diff
@@ -0,0 +1,95 @@
+---
+status: accepted 
+date: 2023-12-19
+deciders: Chris Mungall, Lee Ann 
+consulted: Anastasiya Prymolenna, Brynn Zalmanek, James Tessmer, Montana Smith, Sam Purvine, Yuri Corilo, Mark Miller, Michael Thorton, Alicia Clum, Mark Miller    
+informed: Emiley Eloe-Fadrosh, Shreyas Cholia, Eric Cavanna 
+---
+# Schema Refactoring: Monterey//Berkeley
```

Comment:
Formatting: I don't know whether the double slash (versus a single slash) was intentional.

```diff
- Monterey//Berkeley
+ Monterey/Berkeley
```

Reply:
fixed, will be in push